src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -112,13 +132,23 @@ class SykmeldingService constructor(
                         log.info("Validerer sykmelding mot regler, {}", fields(loggingMeta))
                         val validationResult = regelClient.valider(receivedSykmelding, sykmeldingId)
                         log.info("Resultat: {}, {}", validationResult.status.name, fields(loggingMeta))
+                        if (validationResult.status == Status.OK) {
+
+                            kafkaproducerreceivedSykmelding.send(ProducerRecord(sm2013AutomaticHandlingTopic, receivedSykmelding.sykmelding.id, receivedSykmelding))
+                            log.info("Message send to kafka {}, {}", sm2013AutomaticHandlingTopic, fields(loggingMeta))
+
+                            notifySyfoService(session = session, receiptProducer = syfoserviceProducer, ediLoggId = sykmeldingId,
+                                    sykmeldingId = receivedSykmelding.sykmelding.id, msgId = sykmeldingId, healthInformation = healthInformation)
+                            log.info("Message send to syfoService, {}", fields(loggingMeta))
+                        }
                     }
                 } catch (e: Exception) {
                     PAPIRSM_MAPPET.labels("feil").inc()
                     log.warn("Noe gikk galt ved mapping fra OCR til sykmeldingsformat: ${e.message}, {}", fields(loggingMeta))
                 }
             }
 
+            try {
```

Comment:
Hvis det er meningen at papirmottaket skal skrive til ok-topicen når det går ok må vi passe på at vi ikke oppretter oppgave i tillegg. Oppretting av sak og oppgave her bør vi bare gjøre i de tilfellene der vi enten ikke har nok data i ocr-filen eller hvis noe går galt under prosessering. Her ser det for meg ut som at vi kommer til å opprette oppgave selv om vi har skrevet til topic, og det dermed bør være unødvendig å opprette oppgave.

Reply:
Jepp såg det nå ifix